modules/rosa-create-objects.adoc

```diff
@@ -273,6 +273,81 @@ Create a cluster with autoscaling enabled on the default worker machine pool.
 $ rosa create cluster --cluster-name=mycluster -region=us-east-1 --enable-autoscaling --min-replicas=2 --max-replicas=5
 ----
 
+[id="rosa-create-external-auth-provider_{context}"]
+== create external-auth-provider
+
+Add an external authentication provider instead of an internal OIDC provider to approve users to access your cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create external-auth-provider --cluster=<cluster_name> | <cluster_id> [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--claim-mapping-groups-claim <string>     
+|Required. Describes rules on how to transform information from an ID token into a cluster identity.
+
+|--claim-validation-rule <strings>
+|ClaimValidationRules are rules that are applied to validate token claims to authenticate users. The input will be in a <claim>:<required_value> format. To have multiple claim validation rules, you could separate the values by ','. The input could be in a <claim>:<required_value>,<claim>:<required_value> format.
+
+|--claim-mapping-username-claim <string>
+|The name of the claim that should be used to construct user names for the cluster identity.
```

Comment:
mention the default value

Reply:
Yep it's a required field.

Reply:
Ah, good point, perhaps we could add some valid defaults in a next iteration.

It won't make the ROSA CLI 1.2.38 but it can be a refinement for next one.

Please, @chenz4027, could you follow that suggestion in JIRAs? I think it makes sense.

The usual "defaults" should be something like these ones:
```
--claim-mapping-username-claim=email \
--claim-mapping-groups-claim=groups \
```

Reply:
Sounds good, I created the bug: https://issues.redhat.com/browse/OCM-7673